Glitter-Finance V2 Bridge Update

Elaixzar opened the pull request:
**NOTE**

> - If you would like to add a `volume` adapter please submit the PR [here](https://github.com/DefiLlama/adapters).
> - If you would like to add a `liquidations` adapter, please refer to [this readme document](https://github.com/DefiLlama/DefiLlama-Adapters/tree/main/liquidations) for details.

1. Once your adapter has been merged, it takes time to show on the UI. If more than 24 hours have passed, please let us know in Discord.
2. Please enable "Allow edits by maintainers" while putting up the PR.
3. Sorry, We no longer accept fetch adapter for new projects, we prefer the tvl to computed from blockchain data, if you have trouble with creating a the adapter, please hop onto our discord, we are happy to assist you.
4. Please fill the form below  **only if the PR is for listing a new protocol** else it can be ignored/replaced with reason/details about the PR
5. **For updating listing info** It is a different repo, you can find your listing in this file: https://github.com/DefiLlama/defillama-server/blob/master/defi/src/protocols/data2.ts, you can  edit it there and put up a PR
6. Do not edit/push `package-lock.json` file as part of your changes, we use lockfileVersion 2, and most use v1 and using that messes up our CI
7. No need to go to our discord and announce that you've created a PR, we monitor all PRs and will review it asap

---
##### Name (to be shown on DefiLlama):


##### Twitter Link:


##### List of audit links if any:


##### Website Link:


##### Logo (High resolution, will be shown with rounded borders):


##### Current TVL:


##### Treasury Addresses (if the protocol has treasury)


##### Chain:


##### Coingecko ID (so your TVL can appear on Coingecko, leave empty if not listed): (https://api.coingecko.com/api/v3/coins/list)


##### Coinmarketcap ID (so your TVL can appear on Coinmarketcap, leave empty if not listed): (https://api.coinmarketcap.com/data-api/v3/map/all?listing_status=active,inactive,untracked&start=1&limit=10000)


##### Short Description (to be shown on DefiLlama):


##### Token address and ticker if any:


##### Category (full list at https://defillama.com/categories) *Please choose only one:


##### Oracle used (Chainlink/Band/API3/TWAP or any other that you are using):


##### forkedFrom (Does your project originate from another project):


##### methodology (what is being counted as tvl, how is tvl being calculated):


##### Github org/user (Optional, if your code is open source, we can track activity):

llamatester commented:
The adapter at projects/glitter-finance exports TVL: 
        
 
 
    ethereum                  0
    arbitrum                  0
    polygon_zkevm             0
    avax                      0
    binance                   0
    polygon                   0
    optimism                  0
    algorand                  14.34 k
    solana                    32.74 k
    
    total                    47.07 k

g1nt0ki commented:
hi @Elaixzar  I refactored and merged your code here: https://github.com/DefiLlama/DefiLlama-Adapters/pull/7113

g1nt0ki commented on projects/glitter-finance/index.js:
```diff
@@ -1,9 +1,42 @@
 //addresses
 const SOLANA_VAULT = "7xCU4nvqu3Nz3BBQckKzibp3kBav4xbkuqQ3WM9CBHdJ";
 const ALGO_VAULT = "R7VCOR74LCUIFH5WKCCMZOS7ADLSDBQJ42YURFPDT3VGYTVNBNG7AIYTCQ";
+const ETHEREUM_VAULT = "0xa234acbd98a917f6dda69298e0e7290380006cf1";
```
what is the NOSEC token here?

on other evm chains, it is not a vault but bridged token?

Elaixzar replied:
This is our vault for our bridged for glitter finance.  NOSEC gets bridged to xNOSEC on all the other chains.  THe vault on EThereum holds the bridged NOSEC.  We are in the process of listing the token with coingeko and cmc now.

Elaixzar commented:
> hi @Elaixzar I refactored and merged your code here: #7113

This removed everything I added.  We added vaults for our new token we just released and are in the process of adding to CG and CMC.  

Can you please revert with the vaults included, even though the value currently shows 0, this will not always be the case once the token is added.

g1nt0ki commented:
@Elaixzar just to be clear:

1. are those vaults on other chains or bridged version of NOSEC
2. is NOSEC deployed by your (glitter) team?

Elaixzar commented:
>

Elaixzar commented:
> @Elaixzar just to be clear:
> 
> 1. are those vaults on other chains or bridged version of NOSEC
> 2. is NOSEC deployed by your (glitter) team?

1.  They are vaults, but also bridged version.  How do we reflect we support these chains on our bridge, if we don't add these vaults?  Even if the balance is 0, we are ok with that so long as it reflects we support bridging to those chains.

2.  We support NOSEC on our bridge and hold the TVL in our vaults.

g1nt0ki commented:
1. for bridged tokens, we count it only at source, right now, there is no way for us to show what chains a bridge support, but dont like the idea of listing bridges with 0 tvl.
2. so, totally unrelated teams?

Elaixzar commented:
1.  That's fine.  We will work on adding more vaults on the other chains to show support then.
2.  There is some overlap, but different teams, yes.
